distributions/aws/examples/storage/efs/README.md

```diff
@@ -20,7 +20,10 @@ export CLUSTER_REGION=<clusterregion>
 ```
 
 ## 2.0 Setup EFS
-### **Automated setup**
+
+You can either use Automated or Manual setup 
+
+### 2.1 [Option 1] Automated setup
```

Comment:
Can you mention automated setup is only for dynamic provisioning ?

Reply:
Good catch
Done